Havit/Havit/Screens/Category/View/CategoryIconCollectionViewCell.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  CategoryIconCollectionViewCell.swift
+//  Havit
+//
+//  Created by 김수연 on 2022/01/17.
+//
+
+import UIKit
+import SwiftUI
+
+import SnapKit
+
+class CategoryIconCollectionViewCell: BaseCollectionViewCell {
+
+    // MARK: - property
+
+    private let iconBackgroundView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .gray000
+        view.frame = CGRect(x: 0, y: 0, width: 62, height: 62)
```

Comment:
p2;

18 line에서 init해줘도 좋을 거 같아요!
```suggestion
        let view = UIView(frame: CGRect(x: 0, y: 0, width: 62, height: 62))
```

Reply:
@YoonAh-dev 오호 이렇게도 init 할수 있군ㅇ요! 반영하겠슴돠  반영커밋 : 4364d02